### PR TITLE
fix: bump dependencies

### PR DESCRIPTION
**Pull request by ascheibal**
- owasp 6.5.3
- spring security 5.6.0
- lombok 1.18.22
- liquibase 4.6.2
- spring doc 1.6.0
- mockito 4.1.0
- dgc lib 1.1.8

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -21,24 +21,23 @@
...
-		<mockito.version>3.11.2</mockito.version>
-    	<bcpkix.version>1.69</bcpkix.version>
+		<mockito.version>4.1.0</mockito.version>
+    	<bcpkix.version>1.70</bcpkix.version>
     	<okhttp.version>4.9.1</okhttp.version>
-    	<shedlock.version>4.25.0</shedlock.version>
-        <dgc.lib.version>1.1.7</dgc.lib.version>
-		<sap.cloud.sdk.version>3.57.0</sap.cloud.sdk.version>
+    	<shedlock.version>4.30.0</shedlock.version>
+        <dgc.lib.version>1.1.8</dgc.lib.version>
```
Comment (f11h) [inline pom.xml:38]:
```suggestion
        <dgc.lib.version>1.1.9</dgc.lib.version>
```

**Review by f11h — APPROVED** [review]

File: pom.xml [hunk 2]
```diff
@@ -233,7 +215,7 @@
             <artifactId>scp-cf</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.sap.hcp.cf.logging</groupId>
+            <groupId>com.sap.hcp.cf.ging</groupId>
```
Comment (f11h) [inline pom.xml:218]:
```suggestion
            <groupId>com.sap.hcp.cf.logging</groupId>
```

**Review by f11h — APPROVED** [review]

File: pom.xml [hunk 3]
```diff
@@ -215,7 +215,7 @@
             <artifactId>scp-cf</artifactId>
         </dependency>
         <dependency>
-            <groupId>com.sap.hcp.cf.ging</groupId>
```
Comment (ascheibal) [inline pom.xml]: @f11h : awesome, thanks

Comment (f11h) [reply pom.xml]: How to collect changed lines of code ;)

**Pull request merged**